index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="ja">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>サイト制作課題</title>
+    <link rel="stylesheet" href="./style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Anton&family=Noto+Sans+JP:wght@400;700&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <header class="header">
+      <h1 class="header-logo-wrapper">
+        <img class="header-logo" src="./img/logo.svg" alt="Grand Player" />
+      </h1>
+    </header>
+    <section class="hero">
+      <div class="hero-container">
+        <div class="hero-texts">
+          <p class="hero-main-copy">
+            BE <br />
+            A <br />
+            <span class="hero-main-copy-accent"> "GRAND" </span>
```

Comment:
```suggestion
            <span class="hero-main-copy-accent"> “GRAND” </span>
```
○Figmaのお手本
![スクリーンショット (63)](https://github.com/haru-makes/frontend-basic-site/assets/137466167/8d16d686-5ab0-4457-b786-2be9b3444638)
○作成されたサイト
![スクリーンショット (64)](https://github.com/haru-makes/frontend-basic-site/assets/137466167/5deaaecb-4358-4b87-acd0-18549d3b6949)
凄く分かりにくいのですが、写真の青い所が異なっています。

Reply:
修正しました。